### DISCUSSION TITLE
`necessityIndicator` and `contextualHelp` props are missing in DateRangePicker from react-aria-components

### DISCUSSION THREAD

**Discussion by upender7270** [adobe/react-spectrum#6946] [Q&A] [answered]
I'm developing a date range picker using react-aria-components but we are missing `necessityIndicator` and `contextualHelp` props which are in react-spectrum date range picker.

is this expected? if so how can is add them to labelContext?

**Comment by reidbarber** [comment, marked as answer]
These are features specific to Spectrum, so they aren't included in React Aria components by default, however you can add them to your own usage of React Aria components. For contextual help, just use a [Dialog with Popover](https://react-spectrum.adobe.com/react-aria/Dialog.html#popover). For `necessityIndicator`, you can just add that prop and conditionally render text or an asterisk based on `isRequired`.

**Reply by upender7270** [reply]
Thanks for the info, we have implemented as you said by adding a prop
